[Cleanup] Tooltip On Invoice Number Payment Column

Civolilah opened the pull request:
@beganovich @turbo124 The PR includes the implementation of Ben's recommendation to truncate the `invoice_number` in the Payment column when there are many invoices applied to a specific payment. Screenshot:

![Screenshot 2023-12-16 at 20 42 01](https://github.com/invoiceninja/ui/assets/51542191/533fcad5-6ad9-4cd4-a6a4-fdf9ae3e258a)

Let me know your thoughts.

turbo124 commented:
@Civolilah do we have control over the length before truncate takes over? i think we should show a max of 3-4, and perhaps send the rest into tooltip?

Civolilah commented:
> @Civolilah do we have control over the length before truncate takes over? i think we should show a max of 3-4, and perhaps send the rest into tooltip?

@turbo124 Great! I didn't remember the solution involving a `Tooltip`, but now I've implemented it. Screenshot:

![Screenshot 2023-12-17 at 03 00 35](https://github.com/invoiceninja/ui/assets/51542191/803c463f-3b08-46d2-adfc-48f6fcd6a780)

I did a few more things related to the tooltip here:

- I noticed a bug with the tooltip on the Clients table and Invoice slider, so I fixed it. Screenshot:

![Screenshot 2023-12-17 at 03 17 04](https://github.com/invoiceninja/ui/assets/51542191/868cf7f2-73a3-4375-b64f-706aafa42b3d)

- The tooltip component was not adjusted for `dark mode`, so I just did that now (you probably already noticed the changed background color from the first screenshot).

All changes are very small, so I don't think it will be too much for this small PR.

Let me know your thoughts.
